tests/conftest.py

```diff
@@ -1578,8 +1583,17 @@ def collect_db_dump(request, duthosts):
     yield
     collect_db_dump_on_duts(request, duthosts)
 
-@pytest.fixture(scope="module", autouse=True)
-def verify_new_core_dumps(duthost):
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
Is it better to use "module" level scope?

Reply:
done